lokalutvikling/fp-sak.env

```diff
@@ -0,0 +1,86 @@
+
+OPENIDCONNECT_ISSOHOST=https://vtp:8063/rest/isso/oauth2
+OPENIDCONNECT_ISSOISSUER=https://vtp:8063/rest/isso/oauth2
+OPENIDCONNECT_ISSOJWKS=https://vtp:8063/rest/isso/oauth2/connect/jwk_uri
+OPENIDCONNECT_USERNAME=fpsak-localhost
+
+SYSTEMBRUKER_USERNAME=vtp
+SYSTEMBRUKER_PASSWORD=vtp
+
+#######################################################################################################
+# kafka overstyring
+#######################################################################################################
+KAFKA_TOPIC=privat-foreldrepenger-mottatBehandling-fpsak
+BOOTSTRAP_SERVERS=vtp:9093
+SCHEMA_REGISTRY_URL=skjema-veldet_test
+CLIENT_ID=KP-privat-foreldrepenger-mottatBehandling-fpsak
+KAFKA_AKSJONSPUNKTHENDELSE_TOPIC=privat-foreldrepenger-aksjonspunkthendelse-fpsak
+KAFKA_AKSJONSPUNKTHENDELSE_CLIENT_ID=KP-privat-foreldrepenger-aksjonspunkthendelse
+KAFKA_AKSJONSPUNKTHENDELSE_SCHEMA_REGISTRY_URL=skjema-veldet_test
+KAFKA_KONTROLL_RESULTAT_TOPIC=privat-foreldrepenger-kontroll-resultat-lokal
+KAFKA_KONTROLL_RESULTAT_SCHEMA_REGISTRY_URL=skjema-veldet_test
+KAFKA_KONTROLL_RESULTAT_CLIENT_ID=KC-privat-foreldrepenger-kontroll-resultat
+KAFKA_DOKUMENTBESTILLING_TOPIC=privat-foreldrepenger-dokumenthendelse-vtp
+KAFKA_HISTORIKKINNSLAG_TOPIC=privat-foreldrepenger-historikkinnslag-vtp
+FP_TILKJENTYTELSE_V1_TOPIC_URL=privat-foreldrepenger-tilkjentytelse-v1-local
+KAFKA_RISIKOKLASSIFISERING_TOPIC=privat-foreldrepenger-fprisk-utfor-t4
+KAFKA_RISIKOKLASSIFISERING_CLIENT_ID=KP-privat-foreldrepenger-fprisk
+KAFKA_RISIKOKLASSIFISERING_SCHEMA_REGISTRY_URL=skjema-veldet_test
+FPOPPDRAG_OVERRIDE_PROXY_URL=http://localhost:9000/oppdrag
+FPOPPDRAG_OVERRIDE_DIREKTE_URL=http://vtp:8060/rest/dummy/fpoppdrag
+FPTILBAKE_OVERRIDE_URL=http://vtp:8060/rest/dummy/boolean/false
+DATO_FOR_NYE_BEREGNINGSREGLER=2010-01-01
+
+# Kø
+LDAP_URL=ldaps://vtp:8636/
+LDAP_AUTH=none
+LDAP_USER_BASEDN=ou\=NAV,ou\=BusinessUnits,dc\=test,dc\=local
+MQGATEWAY02_HOSTNAME=e26apvl121.test.local
+MQGATEWAY02_PORT=1411
+MQGATEWAY02_NAME=MUXLSC01
+FPSAK_CHANNEL_NAME=U87_FPSAK_SSL
+SBEH_SAKSBEHANDLING_QUEUENAME=QA.U_SAKOGBEHANDLING.SAKSBEHANDLING
+FPSAK_OKONOMI_OPPDRAG_MOTTAK_QUEUENAME=QA.U82_FPSAK.OKONOMI_OPPDRAG_MOTTAK
+FPSAK_OKONOMI_OPPDRAG_SEND_QUEUENAME=QA.U82_FPSAK.OKONOMI_OPPDRAG_SEND
+RAY_AVSTEM_DATA_QUEUENAME=QA.U82_FPSAK.GRENSESNITTAVSTEMMING_SEND
+
+
+#Propertis for å kjøre FPSAK mot FPFORMIDLING
+FPFORMIDLING_BESTILLBREV_URL=https://host.docker.internal:8010/fpsak/local-development/fpformidling/api/brev/bestill
+FPFORMIDLING_HENTBREVMALER_URL=http://host.docker.internal:8010/fpformidling/api/brev/maler
+FPFORMIDLING_ERDOKUMENTPRODUSERT_URL=http://host.docker.internal:8010/fpformidling/api/brev/dokument-sendt
+FPFORMIDLING_LAGRESAKSBEHANDLERTEKST_URL=http://host.docker.internal:8010/fpformidling/api/saksbehandlertekst/lagre-saksbehandler-tekst
+FPFORMIDLING_HENTSAKSBEHANDLERTEKST_URL=http://host.docker.internal:8010/fpformidling/api/saksbehandlertekst/hent-saksbehandler-tekst
```

Comment:
Disse to kan fjernes - Fpsak master bruker dem ikke lengre. Jeg skal også fjerne dem i Fpsak som en del av opprydningen i TFP-1404. De tre over kommer også til å forsvinne, men trengs foreløpig.